build/docker/ubuntu-trusty/Dockerfile

```diff
@@ -221,3 +221,4 @@ ENV THRIFT_ROOT /thrift
 RUN mkdir -p $THRIFT_ROOT/src
 COPY Dockerfile $THRIFT_ROOT/
 WORKDIR $THRIFT_ROOT/src
+
```

Comment:
This extra line will force an image rebuild.

Reply:
My bad, will revert this as well.